src/hooks/init/init_doctor.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import type { Hook } from '@oclif/core';
+import { Logger } from '@salesforce/core';
+import { VersionCommand } from '@oclif/plugin-version';
+import { Doctor } from '../../doctor';
+
+const log = Logger.childFromRoot('plugin-plugin-info:init_doctor');
+const hook: Hook<'init'> = async ({ config }): Promise<void> => {
+  log.debug('init_doctor hook');
+  if (process.argv[2] === 'doctor') {
```

Comment:
just FYI, `argv` is passed into the init hook, which already has the command name removed. So you could just do this instead,

```typescript
const hook: Hook<'init'> = async ({ config, argv }): Promise<void> => {
  log.debug('init_doctor hook');
  if (argv[0] === 'doctor') {
   // etc...
  }
}
```

Reply:
@mdonnalley what I am seeing in the debugger for argv for command `doctor -d a/b/c/d -c "sfdx org list"` is 

```
Array(4) [-d,
a/b/c/d,
-c,
sfdx org list]
```
I also see the hook signature includes id, which is set to doctor. I am going to go with `id` parameter.